tests/helpers.py

```diff
@@ -142,7 +142,7 @@ def create_pod(
         f"Failed to create resource {pod_name}"
     )
     if wait:
-        assert wait_for_resource_state(pod_obj, desired_status)
+        assert wait_for_resource_state(pod_obj, desired_status, 120)
```

Comment:
@sidhant-agrawal will it be useful to mention timeout=120 so that the context is more clear ? 

Thanks

Reply:
Yes, updated.